openstacksdk_plugin/resources/compute/server.py

```diff
@@ -953,6 +946,40 @@ def _get_user_password(openstack_resource):
             ctx.logger.info('Server has been set with a password')
 
 
+def _disconnect_security_group_from_server_ports(client_config,
+                                                 server_payload,
+                                                 security_group_id):
+    """
+    This method will help to remove connection between port and security group
+    Because when we attach security group to a server that has multiple
+    ports connected to it, all the ports automatically are going to connect
+    to the security group
+    :param dict client_config: Openstack configuration required to connect
+    to API
+    :param dict server_payload: Server payload configuration from openstack
+    :param str security_group_id: Security group ID
+    """
+    if server_payload:
```

Comment:
Why not:
```python
if not server_payload:
  return
```

Reply:
Yeah We can do it on this way, will update it